sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/VectorEmbeddingPolicy.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Constants;
+import com.azure.cosmos.implementation.JsonSerializable;
+import com.azure.cosmos.implementation.apachecommons.lang.StringUtils;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Vector Embedding Policy
+ */
+public final class VectorEmbeddingPolicy {
+
+    private JsonSerializable jsonSerializable;
+
+    /**
+     * Paths for embeddings along with path-specific settings for the item.
+     */
+    @JsonProperty(Constants.Properties.VECTOR_EMBEDDINGS)
+    private List<CosmosVectorEmbedding> cosmosVectorEmbeddings;
+
+    /**
+     * Constructor
+     *
+     * @param cosmosVectorEmbeddings list of path for embeddings along with path-specific settings for the item.
+     */
+    public VectorEmbeddingPolicy(List<CosmosVectorEmbedding> cosmosVectorEmbeddings) {
+        validateEmbeddings(cosmosVectorEmbeddings);
+        this.cosmosVectorEmbeddings = cosmosVectorEmbeddings;
+    }
+
+    /**
+     * Constructor.
+     */
+    public VectorEmbeddingPolicy() {
+        this.jsonSerializable = new JsonSerializable();
+    }
```

Comment:
We have no-args ctor and a ctor that takes cosmosVectorEmbeddings. But I don't see a setter method for setting the embeddings. We should either drop the no-args ctor or the other overload and add the setter method.

Reply:
DOne